pages/_app.js

```diff
@@ -1,14 +1,63 @@
 import GlobalStyle from "@/styles";
 import Head from "next/head";
+import { uid } from "uid";
+import { useRouter } from "next/router";
+import useLocalStorageState from "use-local-storage-state";
 
 export default function App({ Component, pageProps }) {
+  const router = useRouter();
+  const [entries, setEntries] = useLocalStorageState("entries", {
+    defaultValue: [],
+  });
+
+  const [filter, setFilter] = useLocalStorageState("filter", {
+    defaultValue: "all",
+    });
+
+  const [isFavourite, setIsFavourite] = useLocalStorageState("favourite", {
+    defaultValue: "false",
+  });
```

Comment:
Mhh, are you sure you need this state? I can't see where you are using it. And you don't need it to be honest. I think you can remove it and remove `isFavorite` from the being passed down via props as well.